src/map/atcommand.cpp

```diff
@@ -2591,6 +2591,10 @@ ACMD_FUNC(memo)
 		clif_displaymessage(sd->fd,  msg_txt(sd,668)); // Your actual memo positions are:
 		for( i = 0; i < MAX_MEMOPOINTS; i++ )
 		{
+#if PACKETVER_MAIN_NUM >= 20170502 || PACKETVER_RE_NUM >= 20170419 || defined(PACKETVER_ZERO)
+			if (i >= ((MAX_MEMOPOINTS / 2) + pc_readreg2(sd, EXT_MEMO_VAR)))
```

Comment:
MAX_MEMOPOINTS / 2 could cause problems if Gravity extends it even more in the future, consider to leave MAX_MEMOPOINTS as 3 and add a new const instead `MAX_EXT_MEMOPOINTS = MAX_MEMOPOINTS + 3`